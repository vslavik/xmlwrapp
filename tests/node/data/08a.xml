<root>
    <banana/>
    <orange/>
    <apple/>
    <peach/>
    <grape/>
    <strawberry/>
</root>
