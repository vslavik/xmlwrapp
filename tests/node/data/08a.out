<?xml version="1.0"?>
<root>
  <apple/>
  <banana/>
  <grape/>
  <orange/>
  <peach/>
  <strawberry/>
</root>

